asset/snippets/createSavedQuery.js

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//   title: Create SavedQuery
+//   description: Create SavedQuery.
+//   usage: node createSavedQuery <QUERY-ID> <DESCRIPTION>
```

Comment:
@sofisl the [samples style guide](https://googlecloudplatform.github.io/samples-style-guide/#no-cli) says to omit CLIs.

Have we adopted that approach in this repo?

Reply:
@weilin1995 I chatted with folks on this internally - we should **not** use this CLI approach here.

Instead, you'll want to run your sample directly from your tests, like so:
```
// Add this to the bottom of createSavedQuery.js
exports.createSavedQuery = main
```

```
// Add this to your tests
const {createSavedQuery} = require('../createSavedQuery.js');
...
createSavedQuery(...) // Use this to run the sample
```

Reply:
added in the new commit:)